src/Rangeslider.js

```diff
@@ -52,7 +53,8 @@ class Slider extends Component {
     tooltip: true,
     reverse: false,
     labels: {},
-    handleLabel: ''
+    handleLabel: '',
+    keepTooltip: false
```

Comment:
its not working

Reply:
Hi @zhrivodkin 
I tested it like this.
I added `keepTooltip` but it's not working
![image](https://user-images.githubusercontent.com/10303547/43145172-6227129e-8f24-11e8-920c-e056d03f02b1.png)

![image](https://user-images.githubusercontent.com/10303547/43145160-5bf94ee6-8f24-11e8-8dbc-67e4eb9e3174.png)

Reply:
@webmas216 nothing works only with vertical orientation or both?